var/spack/repos/builtin/packages/libunwind/package.py

```diff
@@ -16,6 +16,7 @@ class Libunwind(AutotoolsPackage):
 
     version('develop', branch='master')
     version('2018.10.12', commit='f551e16213c52169af8bda554e4051b756a169cc')
+    version('1.3.1', sha256='43997a3939b6ccdf2f669b50fdb8a4d3205374728c2923ddc2354c65260214f8')
     version('1.3-rc1', sha256='e40f49dcbfdea3f4d15fa555fe68958e69a3f410aacf1ec46fd86aeced699773')
```

Comment:
I think we can remove the release candidate now that a stable release is out. Also, do we want to make this the default version?

Reply:
Re libunwind: I agree it's Ok to remove 1.3-rc1 (I'm the one that
added it).  And I agree make 1.3.1 preferred.

I need to rethink the 2018.10.12 version.  I added it because the
previous stable release 1.2.1 was so hopelessly out of date.  But
going forward, this essentially creates two parallel orders which
spack doesn't handle well.

Reply:
Let's just remove 2018.10.12 then since 1.3.1 is more up-to-date anyway.

Reply:
Not just yet.  My standard packages.yaml for hpctoolkit uses
specifically this rev, so this would immediately break hpctoolkit.
Plus, I haven't tested it yet.

Medium-term, I agree.  But give me a few weeks to check this out and
then I'll submit a followup PR for libunwind.

Reply:
done

Reply:
@iarspider You should move `preferred=True` to the 1.3.1 version
and then remove the 1.3-rc1 version.

Btw, the reason for preferred is that 2018.10.12 > 1.3.1, so without
any preferred setting, 2018.10.12 would be the default version.  But
2018.10.12 is just a snapshot, so it shouldn't be preferred.

Reply:
Ok, I'll sign off on the libunwind changes.
Now all you need is 109 other reviewers. :-)